GitUI/UserControls/FileStatusList.cs

```diff
@@ -48,37 +48,9 @@ public FileStatusList()
             FilterVisible = false;
 
             SelectFirstItemOnSetItems = true;
-            FileStatusListView.MouseMove += FileStatusListView_MouseMove;
-            FileStatusListView.MouseDown += FileStatusListView_MouseDown;
 
-            if (_images == null)
-            {
-                _images = new ImageList
-                {
-                    ImageSize = DpiUtil.Scale(new Size(16, 16)), // Scale ImageSize and images scale automatically
-                    Images =
-                    {
-                        Resources.Removed, // 0
-                        Resources.Added, // 1
-                        Resources.Modified, // 2
-                        Resources.Renamed, // 3
-                        Resources.Copied, // 4
-                        Resources.IconSubmoduleDirty, // 5
-                        Resources.IconSubmoduleRevisionUp, // 6
-                        Resources.IconSubmoduleRevisionUpDirty, // 7
-                        Resources.IconSubmoduleRevisionDown, // 8
-                        Resources.IconSubmoduleRevisionDownDirty, // 9
-                        Resources.IconSubmoduleRevisionSemiUp, // 10
-                        Resources.IconSubmoduleRevisionSemiUpDirty, // 11
-                        Resources.IconSubmoduleRevisionSemiDown, // 12
-                        Resources.IconSubmoduleRevisionSemiDownDirty, // 13
-                        Resources.IconFileStatusUnknown // 14
-                    }
-                };
-            }
-
-            FileStatusListView.SmallImageList = _images;
-            FileStatusListView.LargeImageList = _images;
+            FileStatusListView.SmallImageList = CreateImageList();
+            FileStatusListView.LargeImageList = CreateImageList();
```

Comment:
Previously we had one instance of the image list, now we have two.
Why is the new way better?

Reply:
The implementation uses reference identity to link event callbacks with the property that used the images. It doesn't handle cases where the properties use the same list.